andela/mrm_api
# #164266032 Add structure folder to backend API

#### What does this PR do?
- Add structure folder to backend API
#### Description of Task to be completed?
- add structure folder to backend API
-  create a model to represent office structure levels using Trees
#### How should this be manually tested?
- git pull branch `ft-office-structure-levels-164266032` and check it out
- run `make-build`
- check your database to see if the model is created
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#164266032](https://www.pivotaltracker.com/story/show/164266032)
